ignite/metrics/classification_report.py

```diff
@@ -0,0 +1,121 @@
+import json
+from typing import Callable, Collection, Dict, List, Optional, Sequence, Union
+
+import torch
+
+from ignite.metrics.fbeta import Fbeta
+from ignite.metrics.metric import Metric, reinit__is_reduced
+from ignite.metrics.metrics_lambda import MetricsLambda
+from ignite.metrics.precision import Precision
+from ignite.metrics.recall import Recall
+
+__all__ = ["ClassificationReport"]
+
+
+class ClassificationReport(MetricsLambda):
+    """Build a text report showing the main classification metrics. The report resembles
+        in functionality `https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html#
+        sklearn.metrics.classification_report`_ . The underlying implementation doesn't use the sklearn function.
+
+        Args:
+            beta: weight of precision in harmonic mean
+            output_dict: If True, return output as dict, otherwise return a str
+            output_transform: a callable that is used to transform the
+                :class:`~ignite.engine.engine.Engine`'s ``process_function``'s output into the
+                form expected by the metric. This can be useful if, for example, you have a multi-output model and
+                you want to compute the metric with respect to one of the outputs.
+            device: optional device specification for internal storage.
+            labels: Optional list of label indices to include in the report
+
+        .. code-block:: python
+
+
+            y_true = torch.tensor([[1, 0],
+            [0, 1], [1, 0], [1, 1], [1, 0], [0, 1], [0, 1], [1, 0], [0, 1], [1, 0]])
+            y_pred = torch.randint(0, 2, size=(10,))
+
+            classification_report = ClassificationReport(output_dict=output_dict)
+            classification_report.update((y_true, y_pred))
+            res = classification_report.compute()
+
```

Comment:
Could you please give the output for the example, such that user have an idea of what that does.